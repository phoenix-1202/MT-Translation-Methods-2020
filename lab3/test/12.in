yesIToldYou :: Double -> Int
yesIToldYou x | (x - 5) >= 100500.25 = 12

doNotMergeFunctions :: String -> Int
yesIToldYou s = 5
doNotMergeFunctions a = 1