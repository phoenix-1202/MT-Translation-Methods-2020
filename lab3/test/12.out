Incorrect function name, expected doNotMergeFunctions